File: API/uva/uva.php
<?php 

	$json = file_get_contents('https://uhunt.onlinejudge.org/api/subs-user/888032');

	$json_data = json_decode($json,true);
	include 'C:\xampp\htdocs\Dairy\include\connection.php';
	include 'C:\xampp\htdocs\Dairy\include\tstr.php';

	$pk =array();

	$verdict= array(
	"10" =>"Submission error" , 
	"15" =>"Can't be judged",
	"20" => "In queue",
	"30" => "Compile error",
	"35" => "Restricted function",
	"40" => "Runtime error",
	"45" => "Output limit",
	"50" => "Time limit",
	"60" => "Memory limit",
	"70" => "Wrong answer",
	"80" => "PresentationE",
	"90" => "Accepted");

	// var_dump($json_data);
	$array = $json_data["subs"];

	foreach ($array as $num) {
		$cnt=0;
		$id = $num[0];
		$pid = $num[1];
		$ver = $num[2];
		$tim = $num[4];
		$link ="https://onlinejudge.org/index.php?option=com_onlinejudge&Itemid=8&category=24&page=show_problem&problem=".$pid;
		$pkl="SELECT * FROM uva WHERE id=$pid";
		$n1=$conn->query($pkl);
		$pk2=$n1->fetch_assoc();
		$pname=$pk2['name'];
		$pname2=madestr($pname);

		$sql = "SELECT * FROM piammarajul WHERE subid='$id' AND oj='uva'";
        $result = $conn->query($sql);
        if($result->num_rows==0)
        {
        	$sql = "INSERT INTO piammarajul (subid,dt,link,name,ver,oj)
			VALUES ('$id', '$tim' , '$link' ,'$pname2', '$verdict[$ver]', 'uva')";
			$conn->query($sql);
        }

	}
	
 ?>

File: API/uva/uva.py
import urllib.request, json
import mysql.connector
def uva(name,handel):
	mydb = mysql.connector.connect(
	  host="localhost",
	  user="root",
	  passwd="",
	  database="dairy"
	)
	verdict= dict()
	
	verdict['90']='Accepted'
	verdict['50']='Time Limit Exceeded'
	verdict['70']='Wrong Answer'
	verdict['40']='Runtime Error'
	verdict['30']='Compilation Error'
	verdict['35']='Restricted function'
	verdict['60']='Memory Limit Exceeded'
	verdict['45']='Output Limit'
	verdict['80']='Presentation Error'
	verdict['10']='Submission Error'
	verdict['15']="can not judged"
	verdict['20']='In queue'
	
	mycursor = mydb.cursor()
	url="https://uhunt.onlinejudge.org/api/subs-user/"+handel
	json_url = urllib.request.urlopen('https://uhunt.onlinejudge.org/api/subs-user/888032')
	json_data = json.loads(json_url.read())
	a = json_data["subs"]
	cnt=0
	for row in a:
		vid=row[0]
		cnt+=1;
		ver=verdict[(str)(row[2])]
		time=row[4]
		pid=row[1]
		link="https://onlinejudge.org/index.php?option=com_onlinejudge&Itemid=8&category=24&page=show_problem&problem="+(str)(pid)
	
		mycursor.execute("SELECT * FROM uva WHERE id=%s",(pid,));
		pname=(mycursor.fetchall())[0][2]
		sql1="SELECT * FROM piammarajul WHERE subid=%s AND oj='uva'"
		val1 = (vid,)
		mycursor.execute(sql1,val1)
		myresult = mycursor.fetchall()
		if(len(myresult)==0):
			sql="INSERT INTO piammarajul (subid,dt,link,name,ver,oj) VALUES (%s,%s,%s,%s,%s,'uva')"
			val=(vid,time,link,pname,ver)
			mycursor.execute(sql, val)
			mydb.commit()
		else:
			break
	
	print(cnt)	